react/PromotionBanner.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC } from 'react'
+import PropTypes from 'prop-types'
+import { injectIntl, InjectedIntlProps } from 'react-intl'
+
+import { Button } from 'vtex.styleguide'
+import { usePWA } from 'vtex.store-resources/PWAContext'
+
+import setWebAppData from './utils/webAppIndexedDB'
+
+const propTypes = {
+  imageUrl: PropTypes.string,
+  type: PropTypes.oneOf(['install', 'push-notification']),
+}
+
+const CONSTANTS = {
+  INSTALL_TITLE: 'store/promotion-banner.install.title',
+  INSTALL_DESCRIPTION: 'store/promotion-banner.install.description',
+  INSTALL_ACCEPT_BUTTON_LABEL: 'store/promotion-banner.install.button-accept.label',
+  PUSH_NOTIFICATION_TITLE: 'store/promotion-banner.push-notification.title',
+  PUSH_NOTIFICATION_DESCRIPTION: 'store/promotion-banner.push-notification.description',
+  PUSH_NOTIFICATIONS_ACCEPT_BUTTON_LABEL: 'store/promotion-banner.push-notification.button-accept.label',
+  DISMISS_BUTTON_LABEL: 'store/promotion-banner.button-dismiss.label'
+}
+
+type Props = PropTypes.InferProps<typeof propTypes>
+
+const PromotionBanner: FC<Props & InjectedIntlProps> = ({ intl, type = 'install' }) => {
+  const { showInstallPrompt = null } = usePWA() || {}
```

Comment:
Could it really be null or is it just a TS inconvenience? If so, add a check on [#39](https://github.com/vtex-apps/pwa-components/pull/3/files#diff-5c3386809b77f6d98312956eb41f6355R39)

Reply:
It's just a TS inconvenience 😅